src/API/Helpers/InformationHeaders.php

```diff
@@ -38,62 +50,63 @@ public function setEnvProperty($name, $version)
     }
 
     /**
-     * TODO: Deprecate, not used.
+     * TODO: Deprecate and remove from all dependant libs.
+     *
+     * @param string $name    Dependency or platform name.
+     * @param string $version Dependency or platform version.
      *
-     * @param string $name
-     * @param string $version
+     * @return void
      *
      * @codeCoverageIgnore - Slated for deprecation
      */
     public function setEnvironment($name, $version)
     {
-        $this->data['environment'][] = [
-            'name' => $name,
-            'version' => $version,
-        ];
+        $this->setEnvProperty($name, $version);
```

Comment:
Make sure we're using the correct `env` format here. Deprecated but needs to be refactored in other libs before it can be removed.